ci/scripts/check-no-bazelrc-site.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checks that .bazelrc-site file has not been accidentally committed
+# This file should not be merged upstream
+
+set -e
+if [ -f .bazelrc-site ]; then
```

Comment:
The comment on line 6 says that the script checks whether the file has been committed, but we actually check whether the file exists. Maybe use `git ls-files --error-unmatch .bazelrc-site`?

Reply:
I like the idea of using git here as it is more direct!  Though I do see a couple wrinkles:
1. The command above gives an error on unmatch, not on match. So we'd have to invert the logic
2. At least for my version of git, I don't see a quiet option for this command, so we'd have to do a redirect.

Both of these details would make the script pretty complicated and distinct from the other ci/scripts which are pretty simple to read.

  This script though follows the same assumptions that it is running in a clean repo on a CI host.  This is the same assumption made be many of the other ci/scripts: They are all assumed to be looking at a clean checkout.  

To resolve the confusion though, I've added a comment to this effect to clarify.